test/suites/device-mqtt/main_test.go

```diff
@@ -39,13 +43,6 @@ func TestMain(m *testing.M) {
 	// make sure all services are online before starting the tests
 	utils.WaitPlatformOnline(nil)
 
-	// for local build, the interface isn't auto-connected.
-	// connect manually regardless
-	utils.SnapConnect(nil,
-		"edgexfoundry:edgex-secretstore-token",
-		deviceMqttSnap+":edgex-secretstore-token",
-	)
```

Comment:
The connection should happen here, because the tests depend on it. See other comments.